pkg/config/config_template.yaml

```diff
@@ -145,6 +145,18 @@ api_key:
 # python package.
 # procfs_path: /proc
 
+# The `check_runners` refers to the number of concurrent check runners available
+# for check instance execution. The scheduler will attempt to spread the
+# instances over the collection interval and will _at most_ be running the
+# number of check runners instances concurrently. Setting the value to 1
+# would result in checks running sequentially.
+#
+# We do NOT recommend you change the default number of check runners. Please
+# contact support if you feel you need to modify this setting.
+#
+# check_runners: 4
```

Comment:
actually there already are docs on this option (see 20 lines above), let's update them with what you wrote here :)

Reply:
Updated per your note, clearly missed the option 🤦‍♂️ 

Removed the comment saying 0 would choose the "right" number of runners, as I don't think that was really an accurate statement.